conceptnet5/builders/reduce_assoc.py

```diff
@@ -21,11 +23,91 @@ def concept_is_bad(uri):
             uri.startswith('/a/') or uri.count('/') <= 2)
 
 
-def reduce_assoc(filename, output_filename, cutoff=3, en_cutoff=3):
-    """
-    Takes in a file of tab-separated simple associations, and removes
-    uncommon associations and associations unlikely to be useful.
+class Graph:
```

Comment:
I'd say this class name is overly broad.

Reply:
It started out as something to be used in one file only; now that it is referenced elsewhere I'd have to agree; I'll change it.